packages/edit-site/src/components/layout/router.js

```diff
@@ -133,21 +167,50 @@ export default function useLayoutAreas() {
 		return {
 			key: 'patterns',
 			areas: {
+				sidebar: <SidebarNavigationScreenPatterns />,
 				content: <PagePatterns />,
 				mobile: <PagePatterns />,
 			},
 		};
 	}
 
+	// Styles
+	if ( path === '/wp_global_styles' ) {
+		return {
+			key: 'styles',
+			areas: {
+				sidebar: <SidebarNavigationScreenGlobalStyles />,
+				preview: <Editor isLoading={ isSiteEditorLoading } />,
```

Comment:
`<Editor isLoading={ isSiteEditorLoading } />` gets repeated multiple times, should we move to a constant somewhere?

Reply:
I actually like this personally. It kind of gives clarity about the areas. Ideally we should be passing postId, postType to the Editor component but historically, the active postId/postType are fetched from the site editor store. I think using the store for this is not perfect because the site editor is not just an editor, it can render things that are not "editor" but for now it's like that which means the Editor component is the same between routes (no specific props) while in reality, it's not exactly the same.